CoEpi/ui/symptomReport/FollowUp/Breathless/BreathlessViewController.swift

```diff
@@ -43,15 +43,12 @@ class BreathlessViewController: UIViewController {
         subtitleLabel.text = L10n.Ux.Breathless.subtitle
         skipButtonLabel.setTitle(L10n.Ux.skip, for: .normal)
         
-        
         tableView.register(cellClass: UITableViewCell.self)
         tableView.tableFooterView = UIView()
         tableView.rowHeight = 70.0
         
         tableView.register(UITableViewCell.self, forCellReuseIdentifier: "Cell")
         
-        let images = ["house", "stop", "ground", "hill", "exercise"]
-        
         Observable.just([L10n.Ux.Breathless.p0, L10n.Ux.Breathless.p1, L10n.Ux.Breathless.p2, L10n.Ux.Breathless.p3, L10n.Ux.Breathless.p4])
```

Comment:
This is also data and it should be in the view model.
Also, the translations, images and causes belong together. Orient with e.g. the alerts view model.

Reply:
yep, I moved that in the last commit.